### ISSUE TITLE
Skip time in Practice Mode

### ISSUE THREAD

**Issue by lib-fluff-so** [ShadowMario/FNF-PsychEngine#15973]
### What feature do you want to get added on the **base** engine?

Basically, it would be pretty useful if you could skip time in Practice Mode. Like when you play from chart editor, there is skip time in pause menu. The same can be in practice.

### To test your sight, and reliability, please select the option of what should NOT be requested.

Stage Editor, 6K+ support, and winning icons.

**Comment by Shadow06735** [comment]
i mean you can enable practice mode in charting mode as well

**Comment by lib-fluff-so** [comment]
Yes, you can, but it's better to not enter chart editor to do this. Instead turn practice in week settings, and skip time would be there.
